### PR TITLE
chapter 14 Wm Caton

### PR DESCRIPTION
**Pull request by juliegugge**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal-letters/modernized/15-chapter-14.adoc [hunk 1]
```diff
@@ -118,22 +125,23 @@ or we will not allow any of His messengers and servants to dwell among us,
...
 
+[.offset]
 +++[+++He then proceeds to give a similar narrative of his labors in Friesland, etc.,
 though somewhat more minute than given at this place in the Journal.
-He afterwards adds;]
+He afterwards adds;+++]+++
 
 The sudden and violent storm which you have had in England,^
-footnote:[See the Histories of England concerning the
+footnote:[See the [.book-title]#Histories of England# concerning the
```
Comment (jaredh159) [inline journal-letters/modernized/15-chapter-14.adoc:145]: You did this perfect. 👍

File: journal-letters/modernized/15-chapter-14.adoc [hunk 2]
```diff
@@ -182,7 +190,12 @@ and commit you unto Him,
 who is of power to establish all your hearts in the living Truth, in which I remain,
 your dear brother in the fellowship of sufferings, in the gospel of peace,
 
-William Caton.Amsterdam, 25th of Eleventh month, 1660.
+[.signed-section-signature]
+William Caton.
+
+[.signed-section-closing]
```
Comment (jaredh159) [inline journal-letters/modernized/15-chapter-14.adoc]: This should be a `[.signed-section-context-close]`

File: journal-letters/original/15-chapter-14.adoc [hunk 3]
```diff
@@ -184,7 +197,12 @@ and commit you unto Him,
 who is of power to establish all your hearts in the living Truth, in which I remain,
 your dear brother in the fellowship of sufferings, in the gospel of peace,
 
-William Caton.Amsterdam, 25th of Eleventh month, 1660.
+[.signed-section-signature]
+William Caton.
+
+[.signed-section-closing]
```
Comment (jaredh159) [inline journal-letters/original/15-chapter-14.adoc]: here as well -- should be signed-section-context-close

File: journal-letters/modernized/15-chapter-14.adoc [hunk 4]
```diff
@@ -89,8 +92,12 @@ where we were received with joy and gladness by Friends,
 who rejoiced with us in the good service which we had had;
 and blessed be the Lord for our preservation.
 
+[.offset]
 +++[+++From this city William Caton addressed a letter to Friends in England,
-(as is supposed,) from which the following extracts are selected.]
+(as is supposed,) from which the following extracts are selected.+++]+++
+
+[.embedded-content-document.letter]
+--
```
Comment (jaredh159) [inline journal-letters/modernized/15-chapter-14.adoc:100]: Let's make the `O! My beloved friends` line below a `[.saluation]` -- to match what you did in the original, that seems best to me.  It's a little bit fuzzy whether it's a salutation or not, but I lean towards making it one.

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Good stuff here Julie, just a couple small changes I mentioned in the review comments, and then we'll get this merged.

File: journal-letters/modernized/15-chapter-14.adoc (hunk 1 above)
Comment (juliegugge) [reply journal-letters/modernized/15-chapter-14.adoc:145]: Thanks to the style guide and preview button.... It took me about 17 versions of it to get it to appear correctly! But eventually I hit the nail on the head!

**Review by jaredh159 — APPROVED** [review]

**Comment by jaredh159** [comment]
Awesome, looks great, thanks! Merged.

**Pull request merged**